tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+from bs4 import BeautifulSoup
+from pytest import fixture
+from selenium import webdriver
+
+from data import generate_form_data
+
+
+@fixture
+def set_up_browser():
+    web_driver = webdriver.Chrome()
+    web_driver.maximize_window()
+    yield web_driver
+    web_driver.quit()
+
+
+@fixture
+def get_generated_data():
+    yield generate_form_data()
```

Comment:
Почему `yield`? Мы же возвращаем словарь, над которым далее проводим операции уже в самом тесте, а здесь у нас нет никаких постусловий, которые необходимо будет выполнить.
```suggestion
@fixture
def get_generated_data():
    return generate_form_data()
```